app/src/main/java/com/kickstarter/ui/activities/CommentsActivity.kt

```diff
@@ -128,7 +122,7 @@ class CommentsActivity :
     }
 
     fun postComment(comment: String) {
-        this.viewModel.inputs.postComment(comment, DateTime.now())
+        this.viewModel.inputs.insertNewCommentToList(comment, DateTime.now())
```

Comment:
Could we also call in the `postComment` method  also `Activity.hideKeyboard()` from `com.kickstarter.ui.extensions.ActivityExtKt#hideKeyboard` to hide the keyboard once a comment is posted

Reply:
sure